public/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<head>
+  <meta charset="utf-8" />
+  <meta
+    name="viewport"
+    content="width=device-width, initial-scale=1, shrink-to-fit=no"
+  />
+
+  <title>Ultimate Nike</title>
+</head>
+<body>
+  <div id="app"></div>
+  <div id="photo-gallery"></div>
+  <script src="bundle.js"></script>
+  <script src="http://localhost:3000/bundle.js"></script>
```

Comment:
Is this a duplicate of line 14?

Reply:
no, line 14 is the proxy sever webpack bundle.